pkg/otlp/model/translator/dimensions.go

```diff
@@ -31,9 +31,11 @@ const (
 // Dimensions of a metric that identify a timeseries uniquely.
 // This is similar to the concept of 'context' in DogStatsD/check metrics.
 type Dimensions struct {
-	name string
-	tags []string
-	host string
+	name        string
+	tags        []string
+	host        string
+	originID    string
```

Comment:
I don't think `originID` is needed ✂️

Reply:
I'd rather replace `k8sOriginID` by `originID` though

Reply:
done!